src/Database/LSMTree/Internal/Index/CompactAcc.hs

```diff
@@ -9,14 +10,10 @@
   Incremental construction can be finalised with 'unsafeEnd', which yields both
   a 'Chunk' (possibly) and the `IndexCompact'.
 -}
-module Database.LSMTree.Internal.IndexCompactAcc (
+module Database.LSMTree.Internal.Index.CompactAcc (
     -- * Construction
-    -- $construction-invariants
     IndexCompactAcc (..)
   , new
-  , appendSingle
-  , appendMulti
-  , unsafeEnd
```

Comment:
I think there is value in keeping these exports

Reply:
Please see [my below general comment on specialization](https://github.com/IntersectMBO/lsm-tree/pull/476#issuecomment-2491953851).